certbot/main.py

```diff
@@ -405,6 +406,53 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _cert_path_to_lineage(config):
```

Comment:
This method should probably be in `cert_manager.py` next to `find_duplicative_certs` (https://github.com/certbot/certbot/blob/master/certbot/cert_manager.py#L119). Then, it can use `_search_lineages` which does some error checking and standardization exception throwing.